src/utils/reduce_data_dir.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
`src/utils/reduce_data_dir.sh` is moved to `utils/reduce_data_dir.sh`.
(see https://github.com/espnet/espnet/blob/v.0.3.0/utils/reduce_data_dir.sh)

Please confirm it and also modify your script accordingly (but it seems that you don't use this script, and you can just remove it).